src/debugger/channel/AdapterChannel.js

```diff
@@ -79,6 +79,7 @@ export class AdapterChannel {
     );
     let callback = this._pendingRequestCallbacks[response.id];
     callback(response);
+    delete this._pendingRequestCallbacks[response.id];
```

Comment:
Hmm, you are defining "_pendingRequestCallbacks" as a ES6 map while using it as a raw object map. Is it correct?

Reply:
It works functionality wise and there are no flow errors. I will change it to use ES6 map to be consistent though.